### ISSUE TITLE
foreignKey introduced via postgraphile.tags.json5 is ignored

### ISSUE THREAD

**Issue by btilly** [graphile/postgraphile#1207]
I'm submitting a ... 

* [x] bug report
* [ ] feature request
* [ ] question 

PostGraphile version:  v4.5.0

Minimal SQL file that can be loaded into a clean database:

```sql
create table model.a_base (
    id bigint primary key
);

create table model.a_child1 (
    id bigint primary key,
    a_id bigint not null
);

comment on table model.a_child1 
  is E'@foreignKey (a_id) references model.a_base (id)';

create table model.a_child2 (
    id bigint primary key,
    a_id bigint not null
);
```

Steps to reproduce:

I am running in Docker with the following postgraphile.tags.json5 file (irrelevant parts omitted).  Other parts of this file were paid attention to so I know that it is being properly included.

```
{
    version: 1,
    config: {
        /*
         * There can be entries here for:
         *
         * - `class`: for tables, composite types, views and materialized views
         * - `attribute`: for columns/attributes (of any 'class' type)
         * - `constraint`: for table constraints
         * - `procedure`: for functions/procedures
         */         
        class: {
            "model.a_child2": {
                tags: {
                    foreignKey: "(a_id) references model.a_base (id)"
                }
            }
        }
    }
}
```

Current behavior:
In graphiql it understands that aChild1 has a relationship to aBase in all of the expected cases.  But this is not true for aChild2.

Expected behavior:
The relationship introduced in the postgraphile.tags.json5 for aChild2 should work the same as the one in the database comment for aChild1.

**Comment by benjie** [comment]
You're right; the tags are loaded too late for the `foreignKey` and `primaryKey` smart tags to take effect.

**Comment by benjie** [comment]
This'll probably fix it; but I need to write some tests for it before I can merge it and release it: https://github.com/graphile/graphile-engine/pull/584

**Comment by benjie** [comment]
Fixed in v4.5.4
